product-scenarios/1-manage-public-partner-private-apis/1.1-expose-service-as-rest-api/1.1.2-create-rest-api-by-import-oas-document/src/test/java/org/wso2/am/scenario/tests/rest/api/creation/RESTApiCreationUsingOASDocTestCase.java

```diff
@@ -314,11 +317,14 @@ public void testCreateApiUsingValidOASDocumentFromYamlURL(String url) throws Exc
 
         apiRequest = new APIRequest(import_definition_url, "", swagger_url, type);
 
+        SimpleHTTPServer server = new SimpleHTTPServer();
+        new Thread(server).start();
+
         HttpResponse serviceResponse = apiPublisher.designAPIWithOAS(apiRequest);
+        server.stop();
         Assert.assertTrue(serviceResponse.getData().contains("imported"));
 
-        new Thread(new SimpleHTTPServer()).start();
-
+        new Thread( new SimpleHTTPServer()).start();
```

Comment:
How do we stop this server?

Reply:
This happens at the finally block in the server